functionalTest/test/pageobjects/dashboardPage.js

```diff
@@ -11,7 +10,9 @@ class DashboardPage extends Page {
     return $(".navbar").$(".heading");
   }
   get headlineBanner() {
-    return $('[class="headline-banner col-12"] ');
+    return $(
+      "(//*[@id='root']/div[@class='App']/div[@class='container-fluid'])[1]"
```

Comment:
This one is very layout oriented.     return $(".headline-banner") would be safer

Reply:
Will change selector

Reply:
Changed selector